packages/epetra/src/Epetra_MpiSmpCommData.h

```diff
@@ -54,7 +54,6 @@
 */
 
 class Epetra_MpiSmpCommData : public Epetra_Data {
```

Comment:
I suspect Epetra_MpiSmpCommData should also be removed, since Epetra_MpiSmpComm is being removed.
@maherou can you confirm?

Reply:
@jeforster On closer inspection, I don't see anyplace that Epetra_MpiSmpCommData is used in Trilinos.  Removing it along with Epetra_MpiSmpComm should be fine.  Would you please make that change?  Thanks!

Reply:
@kddevin Makes sense.  I will remove that now.